.github/workflows/chromatic.yml

```diff
@@ -1,22 +1,21 @@
 # .github/workflows/chromatic.yml
 
 # Workflow name
-name: 'Chromatic'
+name: "Chromatic"
 
 # Event for the workflow
 on: push
 
-# List of jobs
 jobs:
   chromatic-deployment:
     # Operating System
     runs-on: ubuntu-latest
-    # Job steps
     steps:
       - uses: actions/checkout@v1
       - name: Install dependencies
         run: npm ci
-        # 👇 Adds Chromatic as a step in the workflow
+      - name: Build react icons
```

Comment:
why is this needed? isn't it enough to to be ran as part of the regular build step in the package.json?

Reply:
@shlomitc It isn't enough - it was crashing on all these steps before

Reply:
this is the only one I see that doesn't have any build actions at all, what command is it running behind the scenes that build the storybook?